parser/parse.go

```diff
@@ -22,3 +21,16 @@ func Parse(doc *goquery.Document, filters []string) (parsed string) {
 	}
 	return
 }
+
+func retriveDescriptionData(doc *goquery.Document, filter string) string {
+	return retriveLastWord(retriveDescriptionFromHTML(doc, filter))
+}
+
+func retriveDescriptionFromHTML(doc *goquery.Document, filter string) string {
+	return strings.TrimSpace(doc.Find(filter).Contents().Get(0).Data)
+}
+
+func retriveLastWord(words string) string {
```

Comment:
Just to make it clear, the idea here is to get the last word only as description or it's related to some parser issue? In case it's "Partly cloudy" will it return "cloudy"?

Reply:
Sorry, I did not think of the possibility of the description being composed of two words like "Partly Cloudy"

Reply:
I suppose that the site has changed the tag structure and now the string "Weather report: Fog" is in the same tag

Reply:
Hmm I see, make sense, yeah in this case I would say the last word is fine, it's enough "cloudy" for example, in general the last word will give the main context :)

Reply:
Thanks for caching it ;)